applications/FluidDynamicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -180,6 +182,21 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
         .def("Execute", &AccelerationLimitationUtilities::Execute)
         ;
 
+    py::class_<MassConservationUtility>(m,"MassConservationUtility")
+    .def(py::init < Model&, Parameters >())
+    .def(py::init< ModelPart&, Parameters >())
+    .def("Initialize", &MassConservationUtility::Initialize)
+    .def("ComputePositiveVolume", &MassConservationUtility::ComputePositiveVolume)
+    .def("ComputeNegativeVolume", &MassConservationUtility::ComputeNegativeVolume)
+    .def("ComputeFlowOverBoundary", &MassConservationUtility::ComputeFlowOverBoundary)
+    .def("ComputeBalancedVolume", &MassConservationUtility::ComputeBalancedVolume)
+    .def("ComputeDtForConvection", &MassConservationUtility::ComputeDtForConvection)
+    .def("ApplyLocalCorrection", &MassConservationUtility::ApplyLocalCorrection)
+    .def("ApplyGlobalCorrection", &MassConservationUtility::ApplyGlobalCorrection)
+    .def("ReCheckTheMassConservation", &MassConservationUtility::ReCheckTheMassConservation)
+    .def("OrthogonalFlowIntoAir", &MassConservationUtility::OrthogonalFlowIntoAir)
```

Comment:
The formulation is conceived for two immiscible fluids (not fluid + air). I'd rename it to something like `OrthogonalFlowFromPositiveToNegativeVolume`.